package.json

```diff
@@ -2881,7 +2881,7 @@
     "integration-test": "node ./out/extension-test/integration/runTests.js",
     "unit-test": "npx mocha --require ts-node/register 'src/extension-test/unit/**/*-test.ts'",
     "unit-test-watch": "npx mocha --watch --require ts-node/register --watch-extensions ts --watch-files src 'src/extension-test/unit/**/*-test.ts'",
-    "prettier-format": "npx prettier --write './**/*.{ts,js,json}'",
+    "prettier-format": "npx prettier --write \"./**/*.{ts,js,json}\"",
```

Comment:
Have you tested if this works on Mac and Linux? I can test that if you don't have a setup for it.

Reply:
I now tried it. Seems to work. Which is a bit surprising. Was the change needed on Windows?

Reply:
HI, yes, this and the watcher doesn't work under windows due to the single quotes, I reverted it for now since it went in prematurely. I plan to do a PR later having tested all occurancies that need changing. Thanks

Reply:
Thanks. It really seemed to work, but much better with a separate PR which can take a broader sweep on this kind of thing.